osi_detectedobject.proto

```diff
@@ -93,16 +93,20 @@ message DetectedObject
     // The estimated probabilities for the object to belong to a specific class.
     //
     optional ClassProbability class_probability = 17;
-
-    // The estimated pedestrian head pose.
+    
+    // Pedestrian head pose for behaviour prediction. Describes the head
+    // orientation w.r.t. the host vehicle orientation.
+    // x-axis of head_pose is pedestrian's straight ahead viewing direction.
     //
-    // \note Pedestrian specific value.
-    optional Orientation3d pedestrian_head_pose = 18;
+    // View_normal_base_coord_system = Inverse_Rotation(head_pose)*Unit_vector_x 
+    optional Orientation3d head_pose = 18;
 
-    // The estimated pedestrian upper body pose.
+    // Pedestrian upper body pose for behaviour prediction. Describes the
+    // upper body orientation w.r.t. the host vehicle orientation.
+    // x-axis of upper_body_pose is pedestrian's upper body intended moving direction.
```

Comment:
Proposed extended definition for reduced ambiguity:
``The x-axis of the right-handed upper body frame is pointing along the pedestrian's upper body intended moving direction and the z-axis is pointing upwards.``

Reply:
// The x-axis of the right-handed upper body frame is pointing along the 
    // pedestrian's upper body ventral direction [2] (i.e. usually pedestrian's 
    // intended moving direction) and the z-axis is pointing upwards (to 
    // pedestrian's head).
    //
    // \par References: 
    // [2] https://en.wikipedia.org/wiki/Anatomical_terms_of_location

Reply:
Since we define attitude by intrinsic rotations, the direction of the z-axis is important! Whether it points upwards or downwards basically determines wheter a positive yawing motion corresponds to a movement of the body towards the left or towards the right.
Good idea to mention the ventral direction!